site/content/blog/hello-sourced.md

```diff
@@ -0,0 +1,92 @@
++++
+date = "2017-11-15T00:00:00-00:00"
+title = "source{d}: why I left Google"
++++
+
+On November 1st I left Google and in my [goodbye note](/blog/googbye/)
+I sneakily said I would "try my luck in a small startup with huge potential".
+
+The time to explain more has come, so let me tell you what I'm up to lately.
+I am now the VP of Developer Relations at [source{d}](https://sourced.tech).
+Maybe you've never heard about it, maybe you've heard a bit, or used some of
+their awesome open source libraries such as
+[go-git](https://github.com/src-d/go-git) or
+[kmcuda](https://github.com/src-d/kmcuda) or
+[proteus](https://github.com/src-d/proteus).
```

Comment:
https://github.com/src-d/go-kallax a good bet as well, been quite popular with the Go folks.